src/test/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+# Root logger option
+log4j.logger.org.fcrepo=INFO,stdout
+#log4j.logger.org.fcrepo.client=DEBUG,stdout
+
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+#log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
```

Comment:
channeling @awoods here: please replace the log4j config with logback format. Here is an example: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/test/resources/logback-test.xml